.github/workflows/ci.yaml

```diff
@@ -4,8 +4,22 @@ on:
     branches: [main]
   pull_request:
   workflow_dispatch:
+  # Run every 6 hours Mon-Fri
+  schedule:
+    - chron: "* */6 * * 1-5"
```

Comment:
when even your CI gets a weekend break 😢

Reply:
:laughing: yep CI costs $$ electricity somewhere and gotta give it a break :wink: but maybe this will be abused and more breaking changes will go in on the weekend :upside_down_face: